AndroidAnnotations/androidannotations-core/androidannotations-test/src/main/java/org/androidannotations/test/ActivityWithServices.java

```diff
@@ -144,9 +146,8 @@
 	// @SystemService
 	// HdmiControlManager hdmiControlManager; // hidden API
 
-	// TODO no support yet in Robolectric
-	// @SystemService
-	// InputManager inputManager;
+	@SystemService
+	InputManager inputManager;
```

Comment:
I think you have to uncomment the [other end](https://github.com/WonderCsabo/androidannotations/blob/robolectric3/AndroidAnnotations/androidannotations-core/androidannotations-test/src/test/java/org/androidannotations/test/ServiceInjectionTest.java#L67) of this test too?